proxies/live/apiproxy/targets/ih-target.xml

```diff
@@ -117,8 +117,11 @@
             <Enabled>true</Enabled>
         </SSLInfo>
         <LoadBalancer>
-            <Server name="immunisation-history"/>
+          <Properties>
```

Comment:
<Properties> should  be a child of the HTTPTargetConnection .. not of the LoadBalancer

Reply:
(Properties that is)